app/assets/stylesheets/decidim/modules/_cards.scss

```diff
@@ -484,7 +484,7 @@ $datetime-bg: $primary;
   color: $anchor-color;
 
   @include breakpoint(small only){
-    word-break: break-all;
+    word-break: normal;
```

Comment:
Small change so it's not really a big deal, but in the future we should try to split work from separate tickets into separate PRs

Reply:
Ah shoot, I thought I'd put that change in a separate PR I made earlier this week, sorry! Must've forgotten to take the change out of staging before I made the commit on this branch, it didn't show up as staged when I ran `git status`, I'll be sure to do a more thorough check next time 😓

Reply:
...oh wait, this is because I forgot to switch back to the main branch before I made the redirect branch 🤦  Sorry I'm so scatterbrained, I'll make sure not to do *that* in the future

Reply:
No worries, I've done this more times than I can count 😅😅😅